backend/src/test/config.ts

```diff
@@ -0,0 +1,8 @@
+const config = {
+  Memory: true,
+  IP: '127.0.0.1',
+  Port: '27017',
+  Database: '',
+};
+
+export default config;
```

Comment:
I feel like it could be good to put this file and `db-handler` in src/utils/test instead, so that the utils are kept together. It's fine either way though!

Reply:
Yes you are right. For this PR I was planning to just set up an example so anyone can refer to it when they make a test. I might leave it like that so that tests are written after this PR is merged. But definitely separate the test cases out like you said when you create the tests.

Reply:
wait that was meant to go under the other comment lmao

Reply:
Yes I think it's a good idea to put it under `src/utils/test` so it's easier to navigate